src/code/z_actor.c

```diff
@@ -655,7 +655,7 @@ void Flags_SetCollectible(GlobalContext* globalCtx, s32 flag) {
     }
 }
 
-void func_8002CDE4(GlobalContext* globalCtx, TitleCardContext* titleCtx) {
+void TitleCard_Reset(GlobalContext* globalCtx, TitleCardContext* titleCtx) {
     titleCtx->durationTimer = titleCtx->delayTimer = titleCtx->intensity = titleCtx->alpha = 0;
 }
```

Comment:
the other function mentioned above is probably more deserving of reset-- this gets called once in a function that initlizes the actor system. TitleCard_Init is probably more appropriate

Reply:
contextual stuff like this is also why we try to tackle documentation as more of a system wide thing. You have to be familiar with the actor system itself to see these nuances to properly name them. Just something to consider as you find stuff you want to work on